nixos/modules/services/monitoring/netdata.nix

```diff
@@ -201,7 +201,9 @@ in {
       serviceConfig = {
         ExecStart = "${cfg.package}/bin/netdata -P /run/netdata/netdata.pid -D -c /etc/netdata/netdata.conf";
         ExecReload = "${pkgs.util-linux}/bin/kill -s HUP -s USR1 -s USR2 $MAINPID";
-        ExecStartPost = ''while [ "$(netdatacli ping)" != pong ]; do sleep 0.5; done'';
+        ExecStartPost = pkgs.writeScript "wait-for-netdata-up" ''
```

Comment:
Still broken...
`systemd[3487291]: netdata.service: Failed at step EXEC spawning /nix/store/fkfp1bbpc8dzyhim5cj48wd9p9gkhk7m-wait-for-netdata-up: Exec format error`

Reply:
should be ok now :-)